src/dm_ls_scf_create.F

```diff
@@ -208,9 +210,15 @@ SUBROUTINE ls_scf_init_read_write_input(input, ls_scf_env, unit_nr)
       CALL section_vals_val_get(ls_scf_section, "REPORT_ALL_SPARSITIES", l_val=ls_scf_env%report_all_sparsities)
       CALL section_vals_val_get(ls_scf_section, "PERFORM_MU_SCAN", l_val=ls_scf_env%perform_mu_scan)
       CALL section_vals_val_get(ls_scf_section, "PURIFICATION_METHOD", i_val=ls_scf_env%purification_method)
+      CALL section_vals_val_get(ls_scf_section, "SIGN_METHOD", i_val=ls_scf_env%sign_method)
       CALL section_vals_val_get(ls_scf_section, "DYNAMIC_THRESHOLD", l_val=ls_scf_env%dynamic_threshold)
       CALL section_vals_val_get(ls_scf_section, "NON_MONOTONIC", l_val=ls_scf_env%non_monotonic)
-      CALL section_vals_val_get(ls_scf_section, "SIGN_SQRT_ORDER", i_val=ls_scf_env%sign_sqrt_order)
+      CALL section_vals_val_get(ls_scf_section, "S_SQRT_METHOD", i_val=ls_scf_env%s_sqrt_method)
+
+      CALL section_vals_val_get(ls_scf_section, "SIGN_SQRT_ORDER", i_val=ls_scf_env%s_sqrt_order, explicit=explicit)
+      IF (.NOT. explicit) THEN
```

Comment:
This fall back might confuse some people as it's not documented. Maybe a single `SQRT_ORDER` keyword is sufficient? Alternatively, I'd propose a third keyword `DEFAULT_SQRT_ORDER`.

Reply:
Up to now SIGN_SQRT_ORDER has been used for the sqrt of the overlap matrix.  
However, I have now included the calculation of the sqrt of the overlap matrix with the pth-root method of Richters which does not consider the sign function. That is why I wanted to rename SIGN_SQRT_ORDER it to S_SQRT_ORDER. However, some of the regtests contain SIGN_SQRT_ORDER and maybe also some in-the-wild input files. Hence, I included the above backward compatibility. If it is ok to just rename parameters in the code and the regtest, I could also simply do that instead.

Reply:
Thomas Kühne has suggested the alias-functionality for the backwards compatibility. I was not aware of this before.

Reply:
Yes, the alias is a a very nice solution!

We don't have any hard rules for backwards compatibility as it's always a judgment call. Changing less prominent keywords, like `SIGN_SQRT_ORDER`, it's generally ok.